packages/shared/prisma/seed.ts

```diff
@@ -14,6 +14,7 @@ import { ModelUsageUnit } from "../src";
 import { getDisplaySecretKey, hashSecretKey } from "../src/server/auth";
 
 const LOAD_TRACE_VOLUME = 10_000;
+const DEMO_USER_ID_NUMBER = 1;
```

Comment:
i'd revert all changes to the seeder if we do not use them ehre, the users are not attached to any project thus not really useful in the seeder for the rest of the application